src/ServiceFabric/ServiceFabric/ChangeLog.md

```diff
@@ -19,6 +19,10 @@
 -->
 ## Upcoming Release
 
+## Version 3.0.2
```

Comment:
Please remove new version 3.0.2 and put all change logs in the upcoming release section

Reply:
updated